examples/src/EmojiApp.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {
+  createComboboxPlugin,
+  createEmojiPlugin,
+  EmojiCombobox,
+  Plate,
+} from '@udecode/plate';
+import { basicNodesPlugins } from './basic-nodes/basicNodesPlugins';
+import { editableProps } from './common/editableProps';
+import { plateUI } from './common/plateUI';
+import { emojiValue } from './emoji/emojiValue';
+import { createMyPlugins, MyValue } from './typescript/plateTypes';
+
+const plugins = createMyPlugins(
+  [...basicNodesPlugins, createComboboxPlugin(), createEmojiPlugin()],
+  {
+    components: plateUI,
+  }
+);
+
+export default () => (
+  <Plate<MyValue>
+    editableProps={editableProps}
+    plugins={plugins}
+    initialValue={emojiValue}
+    onChange={(e) => console.info(e)}
+  >
+    <EmojiCombobox />
```

Comment:
Could you move it into the plugin prop: `renderAfterEditable`?

Reply:
Updated